### PR TITLE
Added article about msgraph changes

### PR DESCRIPTION
**Pull request by mikefrobbins**

### CODE REVIEW

File: docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+### Update-AzAdApplication
+
+- Input type of parameter `InputObject` has been changed from
+  `Microsoft.Azure.Commands.ActiveDirectory.PSADApplication` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphApplication`
+
+- Output type has been changed from `Microsoft.Azure.Commands.ActiveDirectory.PSADApplication` to
+  `System.Boolean`
+
+### Change of Application
```
Comment (dingmeng-xue) [inline docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md:53]:
```suggestion
### Changes to Application Object
```

File: docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+
+- Object `ObjectId` has been replaced by `Id`
```
Comment (dingmeng-xue) [inline docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md:55]:
```suggestion
- `ObjectId` has been replaced by `Id`
```

File: docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+### Update-AzAdServicePrincipal
+
+- Input type of parameter `InputObject` has been changed from
+  `Microsoft.Azure.Commands.ActiveDirectory.PSADServicePrincipal` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphServicePrincipal`
+
+- Output type has been changed from `Microsoft.Azure.Commands.ActiveDirectory.PSADServicePrincipal`
+  to `System.Boolean`
+
+### Changes of Service Principal
```
Comment (dingmeng-xue) [inline docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md:147]:
```suggestion
### Changes to Service Principal Object
```

File: docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+
+- `ApplicationId` has been replaced by `AppId`
+
+- `ObjectType` has been replaced by "OdataType`
```
Comment (dingmeng-xue) [inline docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md:151]:
```suggestion
- `ObjectType` has been replaced by `OdataType`
```

File: docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+### Update-AzAdUser
+
+- Input type of parameter `InputObject` has been changed from
+  `Microsoft.Azure.Commands.ActiveDirectory.PSADUser` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphUser`
+
+- Output type has been changed from `Microsoft.Azure.Commands.ActiveDirectory.PSADUser` to
+  `System.Boolean`
+
+### Changes of User Object
```
Comment (dingmeng-xue) [inline docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md:213]:
```suggestion
### Changes to User Object
```

File: docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md [hunk 6]
```diff
@@ -0,0 +1,294 @@
...
+
+- `ObjectType` has been replaced by "OdataType`
```
Comment (dingmeng-xue) [inline docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md:215]:
```suggestion
- `ObjectType` has been replaced by `OdataType`
```

File: docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md [hunk 7]
```diff
@@ -0,0 +1,294 @@
...
+
+- `ImmutableId` has been replaced by "OnpremisesImmutableId`
```
Comment (dingmeng-xue) [inline docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md:217]:
```suggestion
- `ImmutableId` has been replaced by `OnpremisesImmutableId`
```

File: docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md [hunk 8]
```diff
@@ -0,0 +1,294 @@
...
+
+### Remove-AzAdGroup
+
+- Input type of parameter `InputObject` has been changed from
+  `Microsoft.Azure.Commands.ActiveDirectory.PSADGroup` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphGroup`
+
+### Changes of Group Object
+
+- `ObjectType` has been replaced by "OdataType`
```
Comment (dingmeng-xue) [inline docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md:241]:
```suggestion
- `ObjectType` has been replaced by `OdataType`
```

File: docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md [hunk 9]
```diff
@@ -0,0 +1,294 @@
...
+  `Microsoft.Azure.Commands.ActiveDirectory.PSADGroup` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphGroup`
+
+### Remove-AzAdGroupMember
+
+- Input type of parameter `GroupObject` has been changed from
+  `Microsoft.Azure.Commands.ActiveDirectory.PSADGroup` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphGroup`
+
+## Differences between output from AAD and MSGraph
```
Comment (dingmeng-xue) [inline docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md:268]: The content of that section has been merged into previous sections. This whole section should be deleted.

File: docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md [hunk 10]
```diff
@@ -0,0 +1,294 @@
...
+
+- Parameter `IncludeTotalCount` is not supported and has been removed
+
+### New-AzAdApplication
+
+- Output type has been changed from `Microsoft.Azure.Commands.ActiveDirectory.PSADApplication` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphApplication`
+
+- Parameter `Password` has been removed, customized password is not supported anymore, server will
+  assign secret text when creation
```
Comment (dingmeng-xue) [inline docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md:36]:
```suggestion
- Parameter `Password` has been removed, customized password is not supported anymore, server
  assigns secret text when creation
```

File: docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md [hunk 11]
```diff
@@ -0,0 +1,275 @@
...
+Graph. This change is occurring to ensure a smooth transition in light of the
+[announcement of the retirement of Azure AD Graph](/updates/update-your-apps-to-use-microsoft-graph-before-30-june-2022/).
+
+The following example installs the latest preview version of the `Az.Resources` Azure PowerShell
+module.
+
+```azurepowershell
+Install-Module -Name Az.Resources -Repository PSGallery -Scope CurrentUser -AllowPrerelease
+```
+
```
Comment (dcaro) [inline docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md:24]: For more information, see
[Azure AD to Microsoft Graph migration for Azure command line tools](https://techcommunity.microsoft.com/t5/azure-tools/azure-ad-to-microsoft-graph-migration-for-azure-command-line/ba-p/2836666).

File: docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md [hunk 12]
```diff
@@ -0,0 +1,275 @@
...
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphGroup`
+
+### Remove-AzAdGroupMember
+
+- Input type of parameter `GroupObject` has been changed from
+  `Microsoft.Azure.Commands.ActiveDirectory.PSADGroup` to
+  `Microsoft.Azure.PowerShell.Cmdlets.Resources.MSGraph.Models.ApiV10.IMicrosoftGraphGroup`
+
+For more information, see
+[Azure AD to Microsoft Graph migration for Azure command line tools](https://techcommunity.microsoft.com/t5/azure-tools/azure-ad-to-microsoft-graph-migration-for-azure-command-line/ba-p/2836666).
```
Comment (dcaro) [inline docs-conceptual/azps-6.5.0/azps-msgraph-migration-breaking-changes.md:275]: We could move this line before the list of breaking changes.

**Review by dcaro — APPROVED** [review]
LGTM

**Review by dingmeng-xue — APPROVED** [review]

**Pull request merged**